SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -176,19 +185,28 @@ final class SwedbankPaySDKViewModel: NSObject {
             
             let json = try! JSONEncoder().encode(["paymentorder": paymentOrder])
             
-            var request = try! URLRequest(url: endPointUrl, method: .post, headers: self?.configuration?.headers)
+            var request = URLRequest(url: endPointUrl)// try! URLRequest(url: endPointUrl, method: .post, headers: self?.configuration?.afHeaders)
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 146 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)